.github/workflows/test.yml

```diff
@@ -69,7 +69,7 @@ jobs:
         key: ${{ runner.os }}-go-${{ hashFiles('**/go.sum') }}
         restore-keys: ${{ runner.os }}-go-
     - name: Run Docker containers
-      run: docker-compose up --detach
+      run: docker compose up --detach
```

Comment:
We sure to change to docker compose v2 now?
Will not break any existing functions?
If concern, switch back to v1 if needed.

Reply:
Thanks, if the CI checks pass we should be good peter, this action is just used to spin up the OpenSearch cluster using docker compose and test the provider against that cluster.